src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -453,8 +455,11 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
                   <button
                     className="device-toggle-button"
                     css={audioOptionsOpen ? videoControlActiveStyles : videoControlInActiveStyles}
-                    onClick={() => {
-                      setAudioOptionsOpen(prev => !prev);
+                    onClick={event => {
+                      const target = event.target as Element;
+                      if (!target.closest('#select-microphone')) {
+                        setAudioOptionsOpen(prev => !prev);
+                      }
```

Comment:
prevents the `onclick` event to trigger in the select menu
It would trigger when selecting an option right after setting the `optionsOpen` state of the menu to false, leaving the menu open

Reply:
so you cannot toggle the button anymore? How do you close it?

Reply:
I'll add the code comment to make that clearer
The select menu (`#select-microphone`) is part of the button.
If it's not the target of the click event (when the actual dropdown arrow is clicked), the toggle works normally

Reply:
does this clear things up?
`// We want to ensure to only toggling the menu open or closed when clicking the icon, not the select menu`